src/constants/urls.ts

```diff
@@ -18,3 +20,87 @@ export const TW_IPFS_SERVER_URL = "https://upload.nftlabs.co";
  * @internal
  */
 export const PINATA_IPFS_URL = `https://api.pinata.cloud/pinning/pinFileToIPFS`;
+
+/**
+ * @internal
+ */
+export type ChainOrRpc =
+  | "mumbai"
+  | "polygon"
+  | "rinkeby"
+  | "goerli"
+  | "mainnet"
+  | "fantom"
+  | "avalanche"
+  // ideally we could use `https://${string}` notation here, but doing that causes anything that is a generic string to throw a type error => not worth the hassle for now
```

Comment:
it breaks things like
```typescript
const rpc = "https://...rpc url";
const sdk = new ThirdwebSDK(rpc);
```
^ that's invalid now, because `rpc` is just type: `string`